app/scss/pages/_home.scss

```diff
@@ -15,104 +16,306 @@
+	&__slider {
+		position: relative;	
+		width: 640px;
+		margin: 0 auto;
+		@media (max-width : 680px) {
+			max-width: 100%;
+		}
+	}
+	.testimonial__slider {
+		&-text {
+			font-size: 23px;
+			@media (max-width : 640px) {
+				font-size: 18px;
+			}
+			@media (max-width : 520px) {
+				font-size: 16px;
+			}
+		}
+		&-name {
+			@media (max-width : 520px) {
+				font-size: 14px;
+			}
+		}
 	}
-	&__info {
+	.slick {
+		&-slide {
+			outline: none;
+		}
+		&-arrow {
+			outline: none;
+			border: none;
+			width: 40px;
+			height: 40px;
+			background: #E0B973;
+			cursor: pointer;
+			position: absolute;
+			z-index: 5;
+			border-radius: 8px;
+			top: 95px;
+			@media (max-width : 480px) {
+				display: none;
+			}
+		}
+		&-prev {
+			left: 42%;
+			@media (max-width : 640px) {
+				left: 40%;
+			}
+
+		}
+		&-next {
+			right: 42%;
+			@media (max-width : 640px) {
+				right: 40%;
+			}
+		}
+	}
+}
+
+//footer
+.footer {
+	margin-top: 75px;
+	color: #fff;
+	background-color: #14274A;
+	padding: 65px 0;
+	position: relative;
+	&::after {
+	  content: ''; 
+	  position: absolute;
+	  left: 50%;
+	  transform: translateX(-50%); 
+	  top: -95px;
+		border: 62px solid transparent; 
+		border-bottom: 38px solid #14274A;
+	}
+	&__inner {
 		display: flex;
-		flex-direction: column;
-		justify-content: center;
-		flex: 1;
-		margin-right: 70px;
+		justify-content: space-between;
 		@media (max-width : 990px) {
-			text-align: center;
-			margin-right: 0;
-			order: 2;
-			margin-top: 40px;
-		}
-		&-inner {
-			position: relative;
-			&::before {
-				content: "";
-				border: 1px solid #14274A;
-				position: absolute;
-				left: -35px;
-				height: 100%;
-				@media (max-width : 1170px) {
-					display: none;
-				}
+			flex-wrap: wrap;
+		}
+		@media (max-width : 520px) {
+			flex-direction: column;
+			align-items: center;
+		}
+	}
+	&__item {
+		@media (max-width : 520px) {
+			margin-bottom: 20px;
+		}
+		&--form {
+			@media (max-width : 990px) {
+				margin: 0 auto;
+				text-align: center;
+			}
+			@media (max-width : 672px) {
+				margin-top: 20px;
+			}
+			@media (max-width : 520px) {
+				margin-top: 0;
+			}
+		}
+		&-logo {
+			@media (max-width : 520px) {
+				display: block;
+				margin: 0 auto;
 			}
 		}
-		&-title {
-			font-family: AGaramondPro, sans-serif;
-			font-size: 50px;
-			letter-spacing: 0.02em;
-			font-weight: 700;
-			letter-spacing: 0.02em;
+		&-contacts {
+			margin-top: 20px;
+			font-size: 14px;
 			line-height: 1.1;
-			@media (max-width : 1160px) {
-				font-size: 40px;
+			@media (max-width : 520px)  {
+				text-align: center;
 			}
-			@media (max-width : 990px) {
-				font-size: 35px;
+			.contact__address {
+				margin: 0;
 			}
-			@media (max-width : 480px) {
-				line-height: 1;
+			.contact__phone {
+				display: block;
 			}
 		}
-		&-text {
-			@media (max-width : 480px) {
+	}
+	
+	&-list {
+		list-style: none;
+		font-size: 16px;
+		li {
+			margin-bottom: 10px;
+		}
+	}
+
+	.social__list {
```

Comment:
Check do you use reset css because all `ul` needs `margin:0 padding: 0`

<img width="1393" alt="Screenshot 2021-01-27 at 09 17 00" src="https://user-images.githubusercontent.com/13525860/105956537-a083ba80-6080-11eb-8837-7a7f6c5b0818.png">